src/contexts/events.tsx

```diff
@@ -38,18 +38,27 @@ export interface WalletConnectEvent {
   wallet_action: 'connect' | 'disconnect' | 'change';
 }
 
-export interface TransactionEvent {
-  event: 'transaction_confirmation' | 'kyc_started' | 'kyc_completed' | 'transaction_success' | 'transaction_failure';
+interface OfframpingParameters {
   from_asset: string;
   to_asset: string;
   from_amount: string;
   to_amount: string;
 }
 
+export type TransactionEvent = OfframpingParameters & {
+  event: 'transaction_confirmation' | 'kyc_started' | 'kyc_completed' | 'transaction_success' | 'transaction_failure';
```

Comment:
`transaction_failure` is still contained in this list on purpose so we can use the `createTransactionEvent()` function for the `TransactionFailedEvent` as well